core/src/main/java/io/grpc/internal/AbstractServerImplBuilder.java

```diff
@@ -71,6 +72,11 @@ protected AbstractServerImplBuilder() {
   }
 
   @Override
+  public final T directExecutor() {
+    return executor(MoreExecutors.directExecutor());
```

Comment:
The `newDirectExecutorService()` were used because we used to require an `ExecutorService`.